lib/v2/datasource/remote/model/seeds_history_model.dart

```diff
@@ -0,0 +1,17 @@
+/// The total Number Of Transactions with seeds
+class SeedsHistoryModel {
+  final int totalNumberOfTransactions;
+
+ const SeedsHistoryModel(
+     this.totalNumberOfTransactions,
+  );
```

Comment:
you do not like go in one line all right? 🤣 

` const SeedsHistoryModel(this.totalNumberOfTransactions);`

Reply:
fix 🤣